dev/io.openliberty.microprofile.internal_fat/publish/servers/MPServer/server.xml

```diff
@@ -0,0 +1,8 @@
+<server description="Server for testing microProfile">
```

Comment:
Do we still need this one? Which tests use it?

Reply:
ConfiguredJAXRSCDITest and BasicJAXRSCDITest use this server